### ISSUE TITLE
office.js -  getSharedPropertiesAsync doesnt work when called

### ISSUE THREAD

**Issue by shivani-cloudextend** [OfficeDev/office-js#4836]
Hi Team, 
In Outlook desktop and web application, getSharedPropertiesAsync doesn't work when called to fetch the targetMailbox when using add-in in sharedMailbox.

* Platform: Outlook desktop, Web application for mac and Windows
* Host: Outlook
* Office version number: 16.85.1
* Operating System: mac, windows
* Browser (if using Office on the web): Google Chrome
 
## Expected behavior
getSharedPropertiesAsync should return targetMailbox

## Current behavior
getSharedPropertiesAsync would be called, but no data in it nor the call failed.
There is one error in the network call when a shared mailbox is loaded though.
<img width="567" alt="image" src="https://github.com/user-attachments/assets/348c5e16-a06f-4bb9-bbe8-1d849de2fc62">

## Steps to reproduce
Try to read the details using Office.context.mailbox.item.getSharedPropertiesAsync like below
`Office.context.mailbox.item.getSharedPropertiesAsync(asyncResult => {
            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
              console.log("Shared Properties:", asyncResult.value);
              // Process shared properties
            } else {
              console.error("getSharedPropertiesAsync failed:", asyncResult.error.message);
              // Handle the error
            }
   }`

Could you help us in resolving this issue

**Comment by SE-Hubert** [comment]
Yes this issue was already reported [https://github.com/OfficeDev/office-js/issues/4781](https://github.com/OfficeDev/office-js/issues/4781)

**Comment by rajjha-msft** [comment]
The issue has been fixed and deployed. Please let us know if you still see the issue.

**Comment by sumitjain-rsi** [comment]
in our case, cors and unauthorized error is now not coming. but method  is still not executing.   Office.context.mailbox.item.getSharedPropertiesAsync({}, function (result2) {});
![image](https://github.com/user-attachments/assets/500adc6b-9040-46cf-a568-a07c79af0388)

**Comment by ChibaniMohamed** [comment]
@sumitjain-rsi I'm facing the same issue , did you figure something out ?

**Comment by VSXSYS** [comment]
Same for me

**Comment by shivani-cloudextend** [comment]
@rajjha-msft We are still seeing the issue with the callback of getSharedPropertiesAsync

**Comment by ChibaniMohamed** [comment]
Any updates ?

**Comment by mobisw-msft** [comment]
Hey all, 

We are currently working on this issue and we will keep this thread posted.

Removing `Status:fixed`.

**Comment by AlonEkelund** [comment]
Looking forward to an update and a fix. We have the same problem with two Apps. This has an effect on many of our customers and is highly critical. 

Thank you

**Comment by kawsar-hamid-git** [comment]
Facing the exact same problem. Waiting for the solution.
